avaframe/out3Plot/outDebugPlots.py

```diff
@@ -12,6 +12,14 @@
 import avaframe.out3Plot.plotUtils as pU
 
 
+def plotBufferRelease(inputSimLines, xBuffered, yBuffered):
```

Comment:
please add a short docstring about the purpose

Reply:
done